subprojects/core/src/main/java/org/gradle/api/internal/file/DefaultArchiveOperations.java

```diff
@@ -28,6 +29,16 @@ public DefaultArchiveOperations(FileOperations fileOperations) {
         this.fileOperations = fileOperations;
     }
 
+    @Override
+    public ReadableResource gzip(Object path) {
+        return fileOperations.getResources().gzip(path);
```

Comment:
💄 my Demeter OCD would have me extract `fileOperations.getResources()` to a private `resources()` method :)

Reply:
OCD satisfied :)

Reply:
Thank you! :D